cli/ops/cephadm.py

```diff
@@ -163,7 +163,7 @@ def bootstrap(
         if not nodes:
             raise ResourceNotFoundError("Nodes are required to get mon id")
 
-        kw["mon-ip"] = get_node_ip(nodes, kw.pop("mon-ip"))
```

Comment:
Hey @adityaramteke, I discussed with Vaibhav before sending the fix in this PR, he said it will be OK to club it.
Hey @vamahaja shall I send a separate PR ??

Reply:
Let it be in same PR

Reply:
Okay, thanks @vamahaja